recipes/newrelic/infrastructure/ohi/apache/debian.yml

```diff
@@ -79,21 +79,17 @@ install:
     setup:
       label: "Installing Apache integration..."
       cmds:
-        - |
-          sudo mkdir -p "/etc/newrelic-infra/integrations.d"
         - |
           sudo apt-get update -yq
         - |
-          sudo apt-get install nri-apache -y
+          sudo apt-get install nri-apache -y -qq --no-upgrade
         - |
-          if [ -f /etc/newrelic-infra/integrations.d/apache-config.yml ]; then
-            sudo rm /etc/newrelic-infra/integrations.d/apache-config.yml;
-          fi
+          if [ ! -f /etc/newrelic-infra/integrations.d/apache-config.yml ]; then
```

Comment:
@noahmmcgivern I don't think this is right If I understand correctly.
When re-installing an OHI (apache in this case), we should re-create the apache config file because the user may be inputting new values.
Also if we install a new OHI version (we do), we ll also want to have the latest published config which may contain new elements that the new OHI runtime would be looking for.
I think the intent of the ticket was focused on infra/log, not the OHI configs.
Let's circle back with the PM to clarify.